src/store/sagas/positions.ts

```diff
@@ -760,6 +765,8 @@ export function* handleInitPosition(action: PayloadAction<InitPositionData>): Ge
         })
       )
     }
+
+    yield* call(handleError, error as Error)
```

Comment:
Is it possible for this function not to be repeated every saga?

Reply:
maybe global try catch on all sagas would work but we would have to rethrow that error every try catch in each saga anyway